src/scss/App.scss

```diff
@@ -0,0 +1,184 @@
+$background-color-main: #F4F4F4;
+$background-color-purple: #D0073E;
+$background-color-grey: #EDEDED;
+$text-dark-grey: #919191;
+$border-shadow: #D4D3D3;
+$border-dark-grey: #919191;
+$grey-three: #919191;
+
+@font-face {
+  src: url('../assets/fonts/Abadi-MT-Condensed-Extra-Bold.ttf');
+  font-family: Abadi;
+}
+
+@font-face {
+  src: url('../assets/fonts/AvenirNextLTPro-Cn.otf');
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url('../assets/fonts/nba-spurs.ttf');
+  font-family: spurs;
+}
+
+body {
+  background-color: $background-color-main;
+  margin: 0;
+}
+
+.app {
+  text-align: center;
+}
+
+.active {
+  color: $background-color-purple !important;
```

Comment:
Not good to use `!important` If it's possible avoid and figure out a better way.